packages/components/src/Brand/_docs/Brand.stories.tsx

```diff
@@ -10,12 +11,18 @@ export default {
   parameters: {
     chromatic: { disable: false },
     docs: {
-      description: {
-        component: 'Import { Brand } from "@kaizen/components"',
-      },
+      container: ComponentDocsTemplate,
     },
+    installation: [
+      "npm install @kaizen/Brand",
+      "import { Brand } from `@kaizen/Brand`",
+    ],
+    packageName: "@kaizen/Brand",
+    sourceCodeLink:
+      "https://github.com/cultureamp/kaizen-design-system/tree/main/packages/brand",
+    figmaLink: "Add Figma link here",
   },
-}
+} as Meta<typeof Brand>
 
 export const DefaultStory: StoryFn<typeof Brand> = args => <Brand {...args} />
```

Comment:
We don't need this one anymore, just make a note for what is default in the Variants description.

Reply:
Sweet. This one doesn't have a default as far as I can tell (you always have to pass a `variant` prop). Do I leave it blank in cases like this?

Reply:
Yeah, don't mention a default if there is none 😄